content/logs/log_collection/docker.md

```diff
@@ -152,6 +152,30 @@ LABEL "com.datadoghq.ad.instances"='[{"nginx_status_url": "http://%%host%%:%%por
 LABEL "com.datadoghq.ad.logs"='[{"source": "nginx", "service": "webapp"}]'
 ```
 
+**Java Multiline logs: Docker compose**
+
+For multiline logs like stack traces, the Agent has a [multiline processing rules][9] feature in order to properly aggregate them into a single log.
+
+Example log:
+
+```
+2018-01-03T09:24:24.983Z UTC Exception in thread "main" java.lang.NullPointerException
+        at com.example.myproject.Book.getTitle(Book.java:16)
+        at com.example.myproject.Author.getBookTitles(Author.java:25)
+        at com.example.myproject.Bootstrap.main(Bootstrap.java:14)
+```
+
+Use the `com.datadoghq.ad.logs` label as below on your Java containers to make sure that the above log is properly collected:
+
+  ```
+  labels:
+    com.datadoghq.ad.logs: '[{"source": "java", "service": "myapp", "log_processing_rules": [{"type": "multi_line", "name": "log_start_with_date", "pattern" : "\d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])"}]}]'
+  ```
+
+Check out the [multi-line processing rule documentation][10] to get more pattern examples.
```

Comment:
remove the hyphen in "multi-line" for consistency

Reply:
it seems that consistency is `multi-line`

Reply:
👍